content/about.en.md

```diff
@@ -26,7 +26,7 @@ In early 2014, the French Prime Minister task force for open data, [Etalab](http
 
 The core was improved extensively by Etalab, while the French model was improved and updated by the [_Commissariat général à la stratégie et à la prospective (CGSP)_](https://strategie.gouv.fr/) with the help of the [_Institut d'économie publique (IDEP)_](https://www.idep-fr.org/) and the [_Institut des politiques publiques (IPP)_](https://www.ipp.eu/). A few months later, the French [State Startups incubator](https://beta.gouv.fr) started using the API and extending the model for digital public services purposes, delivering a first consolidated benefits assessment tool targeting citizens, _Mes Aides_, and a payroll simulator.
 
-In 2016, the Open Government Partnership (OGP) Paris Summit demonstrated the international value of OpenFisca when a small team of OpenFisca France maintainers and OpenFisca Tunisia volunteers managed to model Sénégal's income revenue tax in under 36 hours, with a simulator shipped complete with a web UI during the OGP hackathon, winning the team the first prize.
+In 2016, the Open Government Partnership (OGP) Paris Summit [demonstrated](https://www.opengovpartnership.org/wp-content/uploads/2018/07/France_End-of-Term_Report_2015-2017.pdf) the international value of OpenFisca when a small team of OpenFisca France maintainers and OpenFisca Tunisia volunteers managed to [model](https://github.com/openfisca/openfisca-senegal) Sénégal's income revenue tax in under 36 hours, with a simulator shipped complete with a web UI during the OGP hackathon, winning the team the first prize.
```

Comment:
👍 OGP report link.

Reply:
The model link is misleading but the description isn't exactly true. Rephrase it?: `a small team of OpenFisca France maintainers and OpenFisca Tunisia volunteers managed to model Sénégal's income revenue tax`.

As you know, I was part of the OpenFisca Tunisia team (= @benjello and members of [Cahiers de la Liberté](https://www.facebook.com/cahiers.de.la.liberte/)). We updated a bit openfisca_tunisia (+/- taxes and contributions) and developed a web interface on to of it. 

The senegalese model was developed by OpenFisca maintainers and 2 tax inspectors from Senegal. If I remember correctly they also reused the web interface that the Tunisian team developed.

So, regarding the added link here, during OGP at least 2 models were updated/developed: openfisca-senegal and [openfisca-tunisia](https://github.com/openfisca/openfisca-tunisia) (+openfisca-france ? 🤔).

> And we can remove the accents on `Sénégal` in English :)

Reply:
Thanks for these precisions @sandcha! I have to say the specific level of details that make the description “not exactly true” is a bit lost on me, and I fail to see the difference between what I wrote and what you write here. Since that seems important enough for you to not approve this PR, would you mind offering a specific suggestion for the rephrasing you would like to see here? Thank you 🙂